integration-tests/jest-reporter.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash/fp')
+const { parse } = require('path')
+const { createWriteStream, existsSync, mkdirSync } = require('fs')
+
+
+module.exports = class JestReporter {
+  constructor(_globalConfig, _options) {
+    this.logRootDir = process.env.LOG_DIR || '/tmp/test-results'
+  }
+
+  onTestStart(test) {
+    const { path } = test
+    console.info(`\n**  Running ${parse(path).name} at ${this.timeNow()}\n`)
+  }
+
+  onTestResult(_testRunConfig, testResult, _runResults) {
+    const { testFilePath, testResults } = testResult
+    const testName = parse(testFilePath).name
```

Comment:
(Sorry, I missed this in the first round)

We'd typically write this as

```suggestion
    const { name } = parse(testFilePath)
```

and then rename usages of `testName` as `name`. If `testName` is a better name then

```suggestion
    const { name: testName } = parse(testFilePath)
```

would work.

Reply:
Nice! I updated to code to `const { name: testName } = parse(testFilePath)`.